test/inductor/test_torchinductor.py

```diff
@@ -5244,6 +5244,40 @@ def fn(x):
                 assert same(fn(x)[0], compiled([x])[0], equal_nan=True)
                 assert metrics.generated_cpp_vec_kernel_count == 1
 
+        @unittest.skipIf(
+            not codecache.valid_vec_isa_list(), "Does not support vectorization"
+        )
+        @patch("torch.cuda.is_available", lambda: False)
+        def test_vec_compare_op_cpu_only(self):
+            def fn(x):
+                y1 = torch.eq(x, 1)
```

Comment:
Please also add UT: `x == 1`. And the same philosophy is for `ne`, `le`, `ge`, `gt`, `lt`